source/Government.cpp

```diff
@@ -164,6 +164,10 @@ void Government::Load(const DataNode &node)
 		{
 			if(key == "provoked on scan")
 				provokedOnScan = false;
+			else if(key == "reputation max")
+				reputationMax = numeric_limits<double>::max();
+			else if(key == "reputation min")
+				reputationMax = numeric_limits<double>::lowest();
```

Comment:
Notably, the remove behavior would need to be different in order to get this to work as only a grandchild node.